### ISSUE TITLE
Plot > T-x Diagram and P-x Diagram always in mole fraction

### ISSUE THREAD

**Issue by ThukeS** [usnistgov/REFPROP-issues#674]
When either mole or mass basis is chosen in Options > Units > Composition, plots rendered in Plot > T-x Diagram or P-x Diagram are always in mole fraction, however, the x-axis label will reflect the choice. So when mass fraction is chosen, the display is in spurious.

The attached image shows this for the arbitrarily chosen binary R-134a / Propane. The python plots are what I would expect to be rendered.

![image](https://github.com/user-attachments/assets/202cb6cc-3c14-4cd4-97a1-6744e6dc8277)
![mole_v_mass_frac_REFPROP-TPxy](https://github.com/user-attachments/assets/096dce89-4b01-4b2d-b971-041bace43455)

**Comment by ianhbell** [comment]
Yes, this a known bug in the GUI; the plots are always on a molar basis.
